scss/jeet/_settings.scss

```diff
@@ -1,7 +1,5 @@
-/**
- * Grid settings.
- * All values are defaults and can therefore be easily overidden.
- */
+/// Grid settings.
+/// All values are defaults and can therefore be easily overidden.
 
```

Comment:
@corysimmons a description like this would be fine, then?
```scss
/// Sets the calling order when using nested columns sizes, prioritizing the 
/// parent instead of the child.
/// By default, deactivated.
$jeet-parent-first: false !default;
```